include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -760,8 +763,11 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUImage** const _begin,
     const auto assetCount = std::distance(_begin, _end);
     auto res = core::make_refctd_dynamic_array<created_gpu_object_array<asset::ICPUImage> >(assetCount);
 
+    // This should be the other way round because if a queue supports either compute or graphics
+    // but not the other way round
```

Comment:
user must always provide a queue with transfer capability when converting buffers or images

if an explicit compute queue was not provided (even the same one as transfer, i.e. the queue is `nullptr`), because lets say we wont be computing any mip maps, we still need to know the final queue's family index for an ownership transfer.

Setting `computeFamIx==transferFamIx` spares us a lot of checking at the end.